src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$blockSize: 100px;
+$blockGap : 1px;
+$weekDay :
+  "mon" 6,
+  "tue" 5,
+  "wed" 4,
+  "thu" 3,
+  "fri" 2,
+  "sat" 1,
+  "sun" 0;
```

Comment:
but the week should start with monday 🤔

Reply:
it`s values for counting width, and week really starts with monday)))

Reply:
week starts with monday, but not on your calendar )

<img width="1242" alt="Screenshot 2022-07-20 at 22 32 15" src="https://user-images.githubusercontent.com/47606379/180066374-00b2fa28-b8d4-423f-9dad-5c0100097f48.png">

Reply:
It`s ready